verilog/formatting/formatter_test.cc

```diff
@@ -647,6 +647,10 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
      "module baz;\n"
      "endmodule\n"
      "`endif\n"},
+    {"module m;foo bar(.x(^{a,b}));endmodule",
+     "module m;\n"
+     "  foo bar (.x(^{a, b}));\n"
```

Comment:
See tests around line 3083 for various unary/binary operators.

Suggested test cases (for all dual-use operators):
* `{a} ^ {b}` (binary)
* `(a) ^ (b)`
* `a[b] ^ c`
* (you've covered unary `^`)

Reply:
You meant that test: https://github.com/google/verible/blob/906862ac730c4152a0db8ee420000c9601714b5b/verilog/formatting/formatter_test.cc#L3070?

Reply:
Sorry for wrong line number, I meant:

https://github.com/google/verible/blob/906862ac730c4152a0db8ee420000c9601714b5b/verilog/formatting/formatter_test.cc#L448

Reply:
More specifically, cover cases where the operator is on the right as well:
* `}` `^` in `{a} ^ {b}` (but left side could be `)` or `]` too)

(I realize I'm asking for more coverage than the original test case.  My intent is to close off a whole class of similar issues at once.)